app/jitsi/main.py

```diff
@@ -34,3 +35,12 @@ def serve(path):
     if path and not path.startswith('client'):
         return redirect(url_for('main.serve'))
     return send_from_directory(current_app.static_folder, 'index.html')
+
+
+def computeIp():
+    headers_list = request.headers.getlist("X-Forwarded-For")
+    # using the 0th index of headers_list is dangerous for stuff explained here: http://esd.io/blog/flask-apps-heroku-real-ip-spoofing.html 
+    # TODO find a way to make this ALWAYS get client IP
+    user_ip = headers_list[0] if headers_list else request.remote_addr 
```

Comment:
Is `X-Forwarded-For` only present when there's a proxy? I see in the link/StackOverflow that they're also checking `remote_addr` like so:
```
if not request.headers.getlist("X-Forwarded-For"):
   ip = request.remote_addr
else:
   ip = request.headers.getlist("X-Forwarded-For")[0]
```

Reply:
@seangibat yup! Also, when I ran locally, request.headers['X-Forwarded-For'] was always empty, and remote_addr was used. This value was always 127.0.0.1

Reply:
ohhh I'm just blind, I didn't see you're doing a ternary if! my b :smile_cat: